lib/internal/process/main_thread_only.js

```diff
@@ -105,12 +106,10 @@ function setupPosixMethods(_initgroups, _setegid, _seteuid, _nice,
   };
 
   function validateInc(inc) {
-    const tInc = typeof inc;
-    if (tInc === 'number') {
+    if (typeof inc === 'number') {
       validateInt32(inc, 'nice');
-      return _nice(inc);
-    } else if (tInc === 'undefined') {
-      return _nice(inc);
+    } else if (typeof inc === 'undefined') {
```

Comment:
you can just combine this with the below
```js
function validateInc(inc) {
  if (typeof inc === 'number') {
    validateInt32(inc, 'nice');
  } else if (inc !== undefined) {
    throw new ERR_INVALID_ARG_TYPE('nice', ['number', 'undefined'], inc);
  }
}
``

Reply:
Nice catch, totally missed that!
Here it is: 655894e745de20829ff40ee330557da1305f9c7e